src/init.cpp

```diff
@@ -1545,6 +1545,15 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     }
 #endif // ENABLE_MINING
 
+    // Start the thread that notifies listeners of transactions that have been
+    // recently added to the mempool, or have been added to or removed from the
+    // chain.
```

Comment:
It might be useful to add a note here about how this is intentionally created before `step 10: import blocks`.

Reply:
Done.